spec/features/number_of_players_Spec.rb

```diff
@@ -0,0 +1,46 @@
+feature 'can choose to play a single player or two player game' do  
+  scenario 'the player can choose to play a single player game' do
+    visit '/'
+    expect(page).to have_content('To start a single player game')
+  end
+
+  scenario 'the player can choose to play a two player game' do
+    visit '/'
+    expect(page).to have_button('Two Player Game')
+  end
+
+  scenario 'two player names are requested after selecting the two player option' do
+    visit '/'
+    click_button 'Two Player Game'
+    expect(page).to have_field('player_1_name', type: 'text')
+    expect(page).to have_field('player_2_name', type: 'text')    
+  end
+
+  scenario 'the player is presented with player ones choice after entering two names' do
+    start_a_two_player_game
+    expect(page).to have_content('Choose your weapon, JoJo')
```

Comment:
Is it possible that there's something in the background, when you're on Zoom, with 'JoJo' on the cover? 🤔